mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + " years old";
```

Comment:
Hey great work just some code refactoring tip here...

```diff
- return "Hello, my name is " + name + " and I am " + age + " years old";
+ return `Hello my name is ${name} and I am ${age} years old`;
```

- So your solution there works but prone to errors that can be caused **by mistake you can miss on spacing your quotation marks then the words get attached together as one word which is something you did not intend to do** 
- So template literals tend to be the **most effective in concatenating strings etc**

Reply:
Thanks i appreciate